backend/lambdas/process_post/lambda.py

```diff
@@ -0,0 +1,217 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+import json
+import logging
+import os
+import boto3
+import demoji
+import re
+
+from langchain.prompts import PromptTemplate
+from langchain.llms.bedrock import Bedrock
+from langchain_community.chat_models import BedrockChat
```

Comment:
The official package for accessing bedrock models is now [`langchain-aws`](https://github.com/langchain-ai/langchain-aws)

Reply:
Fixed in commit [e611804](https://github.com/aws-samples/ai-powered-text-insights/pull/3/commits/e6118046a160abba4d131ed45237e7d3c6391261)